lib/obs_github_deployments.rb

```diff
@@ -5,7 +5,27 @@
 loader.inflector.inflect("cli" => "CLI")
 loader.setup
 
+require "octokit"
+require "active_support/core_ext/object/blank"
+
 module ObsGithubDeployments
   class Error < StandardError; end
-  # Your code goes here...
+
+  class NoReasonGiven < StandardError
```

Comment:
Before someone comments, I'm going to move the exception classes to a better place :)